x-pack/plugin/src/test/resources/rest-api-spec/test/vectors/10_dense_vector_basic.yml

```diff
@@ -58,17 +58,20 @@ setup:
 
   - match: {hits.total: 3}
 
+  # result should be 65425.624, but we allow 1% error because vectors are indexed as bfloat16
   - match: {hits.hits.0._id: "1"}
-  - gte: {hits.hits.0._score: 65425.62}
-  - lte: {hits.hits.0._score: 65425.63}
+  - gte: {hits.hits.0._score: 62154.34}
```

Comment:
This difference in score is more significant than I expected -- it's closer to 5% than the 1% you mention above. Is the range really so large, or is this just a conservative choice?

Do you have any experimental results on how big the error tends to be (or a maybe just a reference you could link me to)?

Reply:
@jtibshirani Good notice, I made a mistake in calculations instead of multiplying by 0.005, I multiplied by 0.05.  I have adjusted the error rates in the second commit, the error range should be now within 1% . 
 
In [rank_feature field](https://www.elastic.co/guide/en/elasticsearch/reference/master/rank-feature.html), Adrien wrote: "rank_feature fields only preserve 9 significant bits for the precision, which translates to a relative error of about 0.4%.". I have noticed that 0.4% error is not enough for individual vector dimensions, sometimes it is around 0.5%.  Vector function calculations exceed even 0.5%, as an error from each dim can be compounded; 1% seems to be good for small dims vector.   I have not computed an error rate for vector functions on vectors with big dims (100-512), but a good indication that the error rate is acceptable could be that 100% recall was achieved in ann bechmarks for those.